integration-cli/docker_cli_logs_test.go

```diff
@@ -22,8 +22,10 @@ func (s *DockerSuite) TestLogsContainerSmallerThanPage(c *check.C) {
 
 	dockerCmd(c, "wait", cleanedContainerID)
 	out, _ = dockerCmd(c, "logs", cleanedContainerID)
-	if len(out) != testLen+1 {
-		c.Fatalf("Expected log length of %d, received %d\n", testLen+1, len(out))
+	// Each line up to 4096 bytes
+	expectedLen := testLen + testLen/4096 + 1
```

Comment:
`4906` will have to be an exported constant from daemon/logger (e.g. `DefaultBufferSize`)